lekce2/15.py

```diff
@@ -8,7 +8,7 @@
 
 
 word = sys.argv[1]
-upper_indices = [i for i in range(len(word)) if word[i].isupper()] + [len(word)]
+upper_indices = [0] + [i for i in range(len(word)) if word[i].isupper()] + [len(word)]
```

Comment:
Ted tu jen mame dvakrat to same, ale to je fuk :) Jen mozna kdyz uz jsi v tom, zmen u vsech tech verzi volani `.lower()` uvnitr chroustani na volani `.lower()` az na tom vyslednem stringu joinnutem pomoci "_", ono to bude efektivnejsi.